lib/validation/password.rb

```diff
@@ -0,0 +1,9 @@
+module Validation
+  class Password < Validator
+    def validate
+      @errors.push('Password must present') if @value == ''
+      @errors.push('Password must be longer then 6 symbols') if @value.length < 6
+      @errors.push('Password must be shorter then 30 symbols') if @value.length > 30
+    end
```

Comment:
Can we move all the strings to i18n config? From this file and others where you have these 'magic' strings.